Library/Formula/fish.rb

```diff
@@ -36,9 +36,16 @@ def caveats; <<-EOS.undent
       chsh -s #{HOMEBREW_PREFIX}/bin/fish
     to make fish your default shell.
 
-    If you are upgrading from an older version of fish, you should now run:
+    And optionally:
+      fish_config
+    to review configuration.
+
+    If you are upgrading from an older version of fish, you should:
+
+    1. Terminate the outdated fish daemon:
       killall fishd
```

Comment:
Mind just changing this to `killall fishd; fish_update_completions` to get it on one line